include/ck/utility/random_gen.hpp

```diff
@@ -3,6 +3,7 @@
 
 #pragma once
 #include <ck/utility/ignore.hpp>
+#include <ck/utility/type.hpp>
```

Comment:
Thanks! Do we need the entire header here?

Reply:
I guess so, otherwise `bit_cast` is not defined.